src/generate.ts

```diff
@@ -17,10 +17,18 @@ type RequiredModelSubset = Pick<
 	'tables' | 'relationships' | 'synonyms'
 >;
 
+export type Options = {
+	noSerial?: boolean;
```

Comment:
I'd probably rename as
```suggestion
	convertSerialToInteger?: boolean;
```
to be a bit clearer as to what it's doing